src/pages/spawn/spawnVolume/spawnVolumeTable/SpawnVolumeTableActions.tsx

```diff
@@ -1,38 +1,53 @@
+import { useQuery } from "@apollo/client";
 import styled from "@emotion/styled";
 import { size } from "constants/tokens";
+import { MyHostsQuery, MyHostsQueryVariables } from "gql/generated/types";
+import { GET_MY_HOSTS } from "gql/queries";
 import { MyVolume } from "types/spawn";
 import { DeleteVolumeBtn } from "./spawnVolumeTableActions/DeleteVolumeBtn";
 import { EditButton } from "./spawnVolumeTableActions/EditButton";
+import { MigrateBtn } from "./spawnVolumeTableActions/MigrateBtn";
 import { MountBtn } from "./spawnVolumeTableActions/MountBtn";
 import { UnmountBtn } from "./spawnVolumeTableActions/UnmountBtn";
 
 interface Props {
   volume: MyVolume;
 }
 
-export const SpawnVolumeTableActions: React.VFC<Props> = ({ volume }) => (
-  <FlexRow>
-    <DeleteVolumeBtn
-      data-cy={`trash-${volume.displayName || volume.id}`}
-      volume={volume}
-    />
-    {volume.host ? (
-      <UnmountBtn
-        data-cy={`unmount-${volume.displayName || volume.id}`}
+export const SpawnVolumeTableActions: React.VFC<Props> = ({ volume }) => {
+  const { data: myHostsData } = useQuery<MyHostsQuery, MyHostsQueryVariables>(
+    GET_MY_HOSTS
+  );
+  const myHosts = myHostsData?.myHosts ?? [];
+  // Show the migrate button if the volume is a home volume and mounted to a virtual workstation
+  const showMigrateBtn = !!myHosts?.find(
+    (h) => h.homeVolumeID === volume.id && h.distro?.isVirtualWorkStation
+  );
+  return (
+    <FlexRow>
+      <DeleteVolumeBtn
+        data-cy={`trash-${volume.displayName || volume.id}`}
         volume={volume}
       />
-    ) : (
-      <MountBtn
-        data-cy={`mount-${volume.displayName || volume.id}`}
+      {showMigrateBtn && <MigrateBtn volume={volume} />}
```

Comment:
What do you think about hiding the mount/unmount buttons for Virtual Workstations and instead showing just the "Migrate" button? It seems to me like they are basically enabling the same functionality for virtual workstations vs standard hosts (i.e. the ability to move your volume)